db/seeds.rb

```diff
@@ -1,7 +1,84 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+User.create_with(password: 'password').find_or_create_by(email: 'admin@example.org')
```

Comment:
In order to be notified of validation errors, we could use `!` for all the `find_or_create_by`:

```suggestion
User.create_with(
  password: 'password'
).find_or_create_by!(
  email: 'admin@example.org'
)
```